rackcity/views/it_instance_views.py

```diff
@@ -310,14 +314,41 @@ def instance_bulk_upload(request):
             existing_instance = ITInstance.objects.get(
                 hostname=instance_data['hostname'])
         except ObjectDoesNotExist:
-            instances_to_add.append(instance_serializer)
+            model = ITModel.objects.get(id=instance_data['model'])
+            if is_location_full(
```

Comment:
I still think it'd be nice to be able to use validate_location_modification (maybe by a different name) here. If you agree I'm down to change it

Reply:
I agree